api/v1/ytsaurus_types.go

```diff
@@ -452,6 +452,8 @@ type UISpec struct {
 	Theme       string  `json:"theme,omitempty"`
 	Description *string `json:"description,omitempty"`
 	Group       *string `json:"group,omitempty"`
+
+	DirectDownload *bool `json:"directDownload,omitempty"`
```

Comment:
Please add comment above that this option does - it goes into CRD and API doc.

What kind of "direct" is this?
Operations goes via YT http proxy not via UI backend?

Reply:
added a comment and recreated the docs

Reply:
I think the preferable way for operator docs is to have a link to UI code/docs with description what the option does and just mention that operator simply puts field in ui config.

Though if sending PR to UI code seem complicated let's at least describe all options if we choose to use bool pointer — we have to document all three values: false, true, omitted. 
I suppose docs should look like `If omitted or set to true X happend and if set to false — Y happens`.
Also it is not clear what UI is downloading.